src/sagemaker/jumpstart/artifacts.py

```diff
@@ -72,39 +75,36 @@ def _retrieve_image_uri(
         distribution (dict): A dictionary with information on how to run distributed training
         training_compiler_config (:class:`~sagemaker.training_compiler.TrainingCompilerConfig`):
             A configuration class for the SageMaker Training Compiler.
+        tolerate_vulnerable_model (bool): True if vulnerable models should be tolerated (exception
+            not raised). False if these models should raise an exception.
+        tolerate_deprecated_model (bool): True if deprecated models should be tolerated (exception
+            not raised). False if these models should raise an exception.
 
     Returns:
         str: the ECR URI for the corresponding SageMaker Docker image.
 
     Raises:
         ValueError: If the combination of arguments specified is not supported.
+        VulnerableJumpStartModelError: If the model is vulnerable.
```

Comment:
nit: If any of the dependencies required by the script have known security vulnerabilities